src/general/interpolation.jl

```diff
@@ -229,45 +229,14 @@ results = interpolate_plane_3d([0.0, 0.0, 0.0], [1.0, 0.0, 0.0], [0.0, 1.0, 0.0]
 function interpolate_plane_3d(point1, point2, point3, resolution, semi, ref_system, sol;
                               smoothing_length=ref_system.smoothing_length,
                               cut_off_bnd=true, clip_negative_pressure=false)
-    # Verify that points are in 3D space
-    if length(point1) != 3 || length(point2) != 3 || length(point3) != 3
-        throw(ArgumentError("all points must be 3D coordinates"))
-    end
+    coords, resolution_ = sample_plane((point1, point2, point3), resolution)
```

Comment:
This should keep the original resolution

Reply:
if the `resolution` is not a multiple of the edge length, the resolution changes.
The number of points is determined along each edge with `ceil(Int, edge_length/resolution)`.

Reply:
Ah I see this doesn't change anything. Than I don't think this warning is necessary.